packages/hub/test/session.test.ts

```diff
@@ -0,0 +1,117 @@
+import { SessionContext, SessionStatus } from '@sentry/types';
+import { timestampInSeconds } from '@sentry/utils';
+
+import { Session } from '../src/session';
+
+describe('Session', () => {
+  it('initializes with the proper defaults', () => {
+    const session = new Session().toJSON();
+
+    const sessionStartTime = session.timestamp;
+
+    expect(session).toEqual({
+      attrs: {},
+      duration: 0,
+      errors: 0,
+      init: true,
+      sid: expect.any(String),
+      started: expect.any(String),
```

Comment:
It would be interesting to sanity-check the value here to make sure we don't introduce a bug such that we'd forget the multiplication by 1000 (where we have the comment "Make sure that sec is converted to ms for date constructor").

```js
> new Date(Date.now()/1000).toISOString()
"1970-01-19T18:42:13.955Z"
```

For example, we could test that this string starts with the correct year.